saas_app/locale/fr/LC_MESSAGES/django.po

```diff
@@ -25,22 +25,47 @@ msgstr " Approuver"
 msgid "Deny"
 msgstr "Refuser"
 
-#: approve/forms.py:60 approve/forms.py:124 approve/forms.py:194
-#: internal_ops/forms.py:125 internal_ops/forms.py:239
-#: internal_ops/forms.py:333 internal_ops/forms.py:405
+#: approve/forms.py:60 approve/forms.py:127 approve/forms.py:203
+#: internal_ops/forms.py:112 internal_ops/forms.py:231
+#: internal_ops/forms.py:340 internal_ops/forms.py:422
 #: submit_request/forms.py:37 submit_request/forms.py:87
-#: submit_request/forms.py:142 templates/approve/view_all_prev_requests.html:42
+#: submit_request/forms.py:148 templates/approve/view_all_prev_requests.html:42
 #: templates/approve/view_all_requests.html:43
 #: templates/internal_ops/view_all_prev_requests.html:74
 #: templates/internal_ops/view_all_requests.html:75
 #: templates/request/view_all_requests.html:72
 msgid "Cancel"
 msgstr "Annuler"
 
-#: approve/forms.py:90 approve/forms.py:157 internal_ops/forms.py:280
+#: approve/forms.py:90 approve/forms.py:160 internal_ops/forms.py:272
+#: submit_request/forms.py:116
 msgid "Current status"
 msgstr "Statut actuel"
 
+#: approve/forms.py:91 approve/forms.py:161 submit_request/forms.py:117
+msgid "Date manager reviewed the request"
+msgstr "Date à laquelle le responsable a examiné la demande"
+
+#: approve/forms.py:92 approve/forms.py:162 submit_request/forms.py:118
+msgid "Manager approved the request"
+msgstr "Le responsable a approuvé la demande"
+
+#: approve/forms.py:93 approve/forms.py:163 submit_request/forms.py:119
+msgid "Manager denied the request"
+msgstr "Le directeur a refusé la demande"
+
+#: approve/forms.py:164
+msgid "Date sent to S32 approver"
+msgstr "Date d'envoi à l'approbateur S32"
```

Comment:
I'm not 100% sure what's the French word for this? I think it's the right one though :)